.npmignore

```diff
@@ -0,0 +1,2 @@
+.idea/
+.git/
```

Comment:
If you want to be really nice to the users of your lib, exclude everything but the final `index.js`. There is no need they should bundle your readme in their distribution